concepts/randomness/about.md

```diff
@@ -0,0 +1,93 @@
+# Randomness Introduction
+
+Package [`math\rand`][mathrand] provides support for generating pseudo-random numbers.
```

Comment:
```suggestion
Package [math\rand][mathrand] provides support for generating pseudo-random numbers.
```

Although the intention is good, my experience is that the block style might not make it obvious that the text is a link, so it's better to remove it.

Can you fix this in the other places too please?

Reply:
Thanks - done.